doc/source/cluster/kubernetes/examples/tpu-serve-stable-diffusion.md

```diff
@@ -0,0 +1,83 @@
+(kuberay-tpu-stable-diffusion-example)=
+
+# Serve a Stable Diffusion model on GKE with TPUs
+
+> **Note:** The Python files for the Ray Serve application and its client are in the [ray-project/serve_config_examples](https://github.com/ray-project/serve_config_examples). This example is adapted from the [tensorflow/tpu](https://github.com/tensorflow/tpu/tree/master/tools/ray_tpu/src/serve) Cloud TPU example.
+
+## Step 1: Create a Kubernetes cluster with TPUs
+
+Follow [Creating a GKE Cluster with TPUs for KubeRay](kuberay-gke-tpu-cluster-setup) to create a GKE Autopilot or Standard cluster with 1 CPU node and 1 TPU node.
+
+## Step 2: Install the KubeRay operator
+
+Follow [this document](kuberay-operator-deploy) to install the latest stable KubeRay operator via Helm repository. Multi-host TPU support is provided in KubeRay v1.1.0+. Please note that the YAML file in this example uses `serveConfigV2`, which is supported starting from KubeRay v0.6.0.
```

Comment:
It only needs to be installed if the add-on is not enabled.

Reply:
Done in [2a6a93f](https://github.com/ray-project/ray/pull/45924/commits/2a6a93fef8436a802b85f334d779f1b54c4d2d6a).